### DISCUSSION TITLE
Manual qso logger

### DISCUSSION THREAD

**Discussion by dl1igc** [gdyuldin/x6100_gui#31] [Ideas]
I would like to work on a QSO logger for non FT8 QSOs. I think it could be very handy when using the X6100 with external keyboard.
Idea:
- Table with the list of logs. Rotary know to navigate the list and possibility to edit past QSOs.
- Simple edit of QSOs with one line editor (similar to tucnack)
- Export to ADIF.
- Use the sqlite DB?
- Multiple logs?
Feedback welcome

**Comment by gdyuldin** [comment]
Thanks for your suggestion.

My concern is that it might be too complicated for operator and for developement.
For example - it might be implemented using some combinations of keys or just keys to:
* show/hide table
* show new record dialog
* move forward/back through different field of new record
* save/exit from new record dialog
* navigate through table to select record for editing
* delete record from table

So - user should remember all combinations/keys to operate with such qso log. Also - it might be some conflicts with transceiver control keys.
